### ISSUE TITLE
Microsoft Word: reporting of paragraph indenting not in user measurement unit

### ISSUE THREAD

**Issue by michaelDCurran** [nvaccess/nvda#5804]
In MS Word, paragraph indenting is reported as part of the information when pressing NVDA+f (report formatting). However, it is currently only reported in character widths. This is different to what is reported when pressing tab. It should be reported in the measurement unit configured by the user in the MS Word options (E.g. cm, inches etc).

**Comment by nvaccessAuto** [comment]
Incubated in cd3805c4e8e33a1d9f30e8695d3ff7b2966b5e65.
